### PR TITLE
Update celo-blockchain to v1.1.1, attestation service to v1.1.0, add backup_node for chaindata backups

### PR DESCRIPTION
**Pull request by alchemydc**
### Description

Update celo-blockchain to v1.1.1: also upgrades underlying golang.  Removes genesis and bootnode cruft, as this is now handled automatically.

Update attestation service to v1.1.0

Add a backup_node instance type for chaindata backups, which does a nightly tarball backup, and rsync backups every 4 hours.  Previously this was running on the txnode used by the attestation service, which would make attestation unavailable during backups.  This provides higher integrity, fresher backups without any downtime to attestation service.  Backups are stored in GCS in a versioned bucket.

### Other changes

Add scripts for handling generating proof of possessions, cUSD exchanges, etc.

### Tested

On both baklava and mainnet.

### Related issues

n/a

### Backwards compatibility

n/a

### CODE REVIEW

File: packages/terraform-modules-public/gcp/celo-infra/modules/proxy/startup.sh [hunk 1]
```diff
@@ -416,6 +413,7 @@ ExecStart=/usr/bin/docker run \\
       --proxy.proxy \\
       --proxy.proxiedvalidatoraddress ${validator_account_address} \\
       --proxy.internalendpoint :30503 \\
+      --light.serve 0 \\
```
Comment (tkporter) [inline packages/terraform-modules-public/gcp/celo-infra/modules/proxy/startup.sh:358]: curious why this?

**Review by tkporter — APPROVED** [review]
@jcortejoso ptal if you have the time too :)

Looks great! The scripts look super useful. I'm curious why you set `--light.serve 0` though-- were you seeing a performance hit by serving light clients? Would be nice to see it configurable via a var

File: packages/terraform-modules-public/gcp/celo-infra/modules/proxy/startup.sh (hunk 1 above)
Comment (alchemydc) [reply packages/terraform-modules-public/gcp/celo-infra/modules/proxy/startup.sh:358]: Consensus was that validator proxies shouldn't be serving light clients.  See https://discord.com/channels/600834479145353243/697229909545713775/788845333052260405

**Comment by alchemydc** [comment]
Updated blockchain to v1.2.3 and fixed some bugs in the syslog and backup/restore code.

**Review by tkporter — APPROVED** [review]

**Pull request merged**